garage/envs/wrappers/gray_scale.py

```diff
@@ -0,0 +1,35 @@
+"""Grayscale wrapper for gym.Env."""
+import gym
+from gym.spaces import Box
+import numpy as np
+from skimage import color
+
+
+class GrayScale(gym.Wrapper):
+    """Converting frames to grayscale."""
+
+    def __init__(self, env):
+        """
+        Grayscale wrapper.
```

Comment:
Please make this description more explicit.

Reply:
I feel that class docstring will always contain the same content for __init__ docstring. From [google styleguide](http://google.github.io/styleguide/pyguide.html), there is no docstring in __init__. Should we skip the flake8 checking for __init__ docstring?

Reply:
The Sphinx example for Google docstring style says you can document `__init__` args in either the class docstring or the `__init__` docstring, but to only do one of those and be consistent.


Let's go with only doing it in the class. This means you can edit `setup.cfg` and `scripts/travisci/check_flake8.sh` to tell flake8 to ignore [D107](http://www.pydocstyle.org/en/2.1.1/error_codes.html).

I'd appreciate it if you made a PR on https://github.com/rlworkgroup/akro/blob/master/setup.cfg to update it there as well.